scheduler/src/cook/kubernetes/api.clj

```diff
@@ -414,6 +418,13 @@
     (.setEffect toleration "NoSchedule")
     toleration))
 
+(def toleration-for-deletion-candidate-of-autoscaler
+  (let [^V1Toleration toleration (V1Toleration.)]
+    (.setKey toleration "DeletionCandidateOfClusterAutoscaler")
+    (.setOperator toleration "Exists")
+    (.setEffect toleration "PreferNoSchedule")
+    toleration))
```

Comment:
Nitpick: a good place to use [doto](https://clojuredocs.org/clojure.core/doto)

Reply:
Good call, will do

Reply:
Done